Cinema.BLL/EmailSender/MailjetEmailSender.cs

```diff
@@ -0,0 +1,53 @@
+using Mailjet.Client.TransactionalEmails;
+using Microsoft.Extensions.Configuration;
+
+namespace Cinema.BLL.EmailSender;
+using Mailjet.Client;
+using Mailjet.Client.Resources;
+using Newtonsoft.Json.Linq;
+
+
+public class MailjetEmailSender
+{
+    private readonly string? _apiKey;
+    private readonly string? _apiSecret;
+
+    public MailjetEmailSender(IConfiguration configuration)
+    {
+        _apiKey = configuration["Mailjet:ApiKey"];
+        _apiSecret = configuration["Mailjet:ApiSecret"];
+    }
+
+    public async Task SendEmailAsync(string toEmail, string subject, string body)
+    {
+
+        MailjetClient client = new MailjetClient(_apiKey, _apiSecret);
+        MailjetRequest request = new MailjetRequest
+            {
+                Resource = Send.Resource,
+            }
+            .Property(Send.FromEmail, "yaroslavkuzenko02@gmail.com")
+            .Property(Send.FromName, "Yaroslav Kuzenko")
+            .Property(Send.Subject, subject)
+            .Property(Send.TextPart, "Dear passenger, welcome to Mailjet! May the delivery force be with you!")
+            .Property(Send.HtmlPart, "<h3>Dear passenger, welcome to <a href=\"https://www.mailjet.com/\">Mailjet</a>!<br />May the delivery force be with you!")
```

Comment:
I'm not sure if these messages are the best in this case. I doubt "our users" have something to do with Mailjet itself. If were you, I would use the body parameter here.